app/src/test/kotlin/fr/free/nrw/commons/upload/UploadMediaDetailAdapterUnitTest.kt

```diff
@@ -247,4 +247,16 @@ class UploadMediaDetailAdapterUnitTest {
         verify(adapterView, times(3)).adapter
     }
 
+    @Test
+    fun testRemoveTrailingWhitespace() {
+        val test1 = "test  "
```

Comment:
Thanks a lot for the unit test too! Would you mind testing a few more whitespace characters, for instance tab, carriage return, Japanese whitespace? And also without whitespace. Thanks!